backend/src/Repository/CaptainFinancialDuesEntityRepository.php

```diff
@@ -86,4 +87,21 @@ public function getCaptainFinancialDuesByCaptainId(int $captainId): array
 
             ->getResult();
     }
+
+    public function getSumCaptainFinancialDuesById(int $id): array
+    {
+        return $this->createQueryBuilder('captainFinancialDuesEntity')
+
+            ->select('sum( captainFinancialDuesEntity.amount) as sumCaptainFinancialDues')
+
+            ->andWhere('captainFinancialDuesEntity.id = :id')
+
+            ->setParameter('id', $id)
+            
+            ->orderBy('captainFinancialDuesEntity.id', 'DESC')
```

Comment:
As long as you return only the SUM of single column, you do not need to the orderBy statement.